src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,57 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
+import static seedu.addressbook.common.Messages.MESSAGE_USING_STORAGE_FILE;
+
+public class Formatter {
+
+    /** A decorative prefix added to the beginning of lines printed by AddressBook */
+    private static final String LINE_PREFIX = "|| ";
+
+    /** A platform independent line separator. */
+    private static final String LS = System.lineSeparator();
+
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+    /** Offset required to convert between 1-indexing and 0-indexing.  */
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+
+    public static String getStorageFileInfo(String storageFilePath) {
+        return String.format(MESSAGE_USING_STORAGE_FILE, storageFilePath);
+    }
+
+    public static String formatMessage(String... message) {
```

Comment:
Good job.  Having comments explaining the role of formatMessage will help the reader more to understand it.